lnrpc/rpc.swagger.json

```diff
@@ -4097,6 +4101,20 @@
     "lnrpcStopResponse": {
       "type": "object"
     },
+    "lnrpcTimestampError": {
+      "type": "object",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "format": "uint64",
+          "description": "The unix timestamp in seconds when the error occurred."
+        },
+        "error": {
+          "type": "string",
+          "description": "The error string of the error our peer sent us."
```

Comment:
Maybe something like this: "String representation/value of the error the peer sent us"